test/e2e/tekton_test.go

```diff
@@ -55,7 +55,7 @@ func TestTektonPipeline(t *testing.T) {
 	_, err = kubectl.Run("apply", "-f", basedir+"/kn-deployer-rbac.yaml")
 	assert.NilError(t, err)
 
-	_, err = kubectl.Run("apply", "-f", basedir+"/buildah.yaml")
+	_, err = kubectl.Run("apply", "-f", "https://raw.githubusercontent.com/tektoncd/catalog/master/buildah/buildah.yaml")
```

Comment:
did you really want to fix location for this YAML?

Reply:
yes, I had to change and vendor it as the catalog task didnt support docker `FORMAT` param, now it does so removed it.

Reply:
We're already using a fixed location for the `kn` task  (a few lines below). If we want to keep it compatible with the upstream catalog then pointing to the catalog directly makes sense (instead of copying over the files from there).